src/lib/swapper/swappers/ThorchainSwapper/utils/addAggregatorAndDestinationToMemo.test.ts

```diff
@@ -0,0 +1,59 @@
+import { CHAIN_NAMESPACE } from '@shapeshiftoss/caip'
+import { describe, expect, it } from 'vitest'
+
+import { addAggregatorAndDestinationToMemo } from './addAggregatorAndDestinationToMemo'
+import { MEMO_PART_DELIMITER } from './constants'
+
+const RECEIVE_ADDRESS = '0x32DBc9Cf9E8FbCebE1e0a2ecF05Ed86Ca3096Cb6'
+const AGGREGATOR_ADDRESS = '0xd31f7e39afECEc4855fecc51b693F9A0Cec49fd2'
+const FINAL_ASSET_ADDRESS = '0x8a65ac0E23F31979db06Ec62Af62b132a6dF4741'
+const AGGREGATOR_TWO_LAST_CHARS = 'd2'
+const FINAL_ASSET_TWO_LAST_CHARS = '41'
+
+describe('addAggregatorAndDestinationToMemo', () => {
+  it('should add aggregator address, destination address and minAmountOut correctly', () => {
+    const minAmountOut = '9508759019'
+    const affiliateBps = '100'
+    const finalExpectedAmountOut = '9413671428'
```

Comment:
Can we add some L1 terminology and also add a comment explaining that this is irrelevant in the context of those tests? e.g:
```suggestion
      const expectedL1AmountOut = '42' // we don't care about this for the purpose of tests
```

Reply:
Changed it in both addAggregatorAndDestinationToMemo` and `addSlippageToMemo`